config/locales/en.yml

```diff
@@ -478,7 +478,7 @@ en:
     suspended_error:
       heading_html: Your account is under review
       body: >
-        We noticed some unusual activity on your account. We've suspended access while we take a closer look.
+        We've suspended your account because it looks like you have violated the community tipping guidelines. Please read the guidelines <a href="https://community.brave.com/t/a-note-to-publishers/48733">here</a>.
```

Comment:
Would prefer `We noticed some unusual activity on your account. We've suspended access while we take a closer look. This may include violations of the BAT grant guidelines found <a href="https://community.brave.com/t/a-note-to-publishers/48733">here</a>`

Reply:
Let's say 
`This may include violations of the <a href="https://community.brave.com/t/a-note-to-publishers/48733"> BAT grant guidelines.</a>`